src/cms/models/feedback/feedback.py

```diff
@@ -15,6 +15,7 @@ class Feedback(models.Model):
     )
     emotion = models.CharField(max_length=3, choices=EMOTION)
     comment = models.CharField(max_length=1000)
+    isTechnicalFeedback = models.BooleanField(default=False)
     readStatus = models.BooleanField(default=False)
```

Comment:
```suggestion
    is_technical = models.BooleanField(default=False)
    read_status = models.BooleanField(default=False)
```
The [official Django coding styleguides](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style) recommend lowercase & underscore for field names instead of CamelCase.